draft-ietf-jsonpath-base.md

```diff
@@ -150,7 +150,7 @@ from a JSON value.
 
 JSON {{-json}} is a popular representation
 format for structured data values.
-JSONPath defines a string syntax for selecting and extracting JSON values
+JSONPath defines string expressions for selecting and extracting JSON values
```

Comment:
Hmm, this loses that this is a new syntax, specifically for JSONPath.
"expressions" is very general here, which raises more questions than it answers.
I like the old sentence much better in this place.

Reply:
A fair point. Not sure how to firm up the definition of the term "query", but let's address that separately.